src/Behat/Resources/config/services/contexts/setup.yml

```diff
@@ -10,43 +10,36 @@ services:
 
     _defaults:
         public: true
+        autowire: true
 
-    app.behat.context.setup.admin_security:
-        class: App\Behat\Context\Setup\AdminSecurityContext
+    App\Behat\Context\Setup\:
+        resource: '../../../../Context/Setup/*'
+        autowire: true
+        public: true
+
+    App\Behat\Context\Setup\AdminSecurityContext:
         arguments:
-            - "@app.behat.shared_storage"
-            - "@app.behat.admin_security"
-            - "@app.fixture.example_factory.admin_user"
-            - "@sylius.repository.admin_user"
+            $securityService: "@app.behat.admin_security"
+            $userRepository: "@sylius.repository.admin_user"
```

Comment:
It should get autowired with ResourceBundle 1.4+ and the variable name `$adminUserRepository`.

Reply:
@pamil I didn't see this awesome [PR](https://github.com/Sylius/Sylius/pull/10062). Thanks a lot!
I've got to upgrade sylius bundles & components first :)